src/viur/core/modules/file.py

```diff
@@ -969,6 +1153,25 @@ def mark_for_deletion(self, dlkey: str) -> None:
 
         db.Put(fileObj)
 
+    def create_serving_url(self, skel: SkeletonInstance) -> SkeletonInstance:
+        """ Create Serving url for public image files
+        """
+        # try to create a servingurl for images
+        if not conf.instance.is_dev_server and skel["public"] and skel["mimetype"] \
+                and skel["mimetype"].startswith("image/") and not skel["serving_url"]:
+
+            try:
+                bucket = File.get_bucket(skel['dlkey'])
+                skel["serving_url"] = images.get_serving_url(
+                    None,
+                    secure_url=True,
+                    filename=f"/gs/{bucket.name}/{skel['dlkey']}/source/{skel['name']}",
+                )
+            except Exception as e:
```

Comment:
Why is it caught like this and not reraised or another return value used?

Reply:
The intention behind this was that an error during generation would not prevent saving.

Reply:
But how do I know if everything was OK or not when I call this method if neither an exception is raised nor a suitable return value is returned?

Reply:
The `logging.exception(e)` is enough for now.